sdcm/scylla_bench_thread.py

```diff
@@ -183,7 +184,9 @@ def create_stress_cmd(self, stress_cmd, loader, cmd_runner):
                     # server_names = ' '.join(f'-tls-server-name {ip}' for ip in ips.split(","))
                     # stress_cmd = f'{stress_cmd.strip()} -tls-host-verification {server_names}'
 
-        return stress_cmd
+        if is_shell_supported:
+            return stress_cmd
+        return stress_cmd.removeprefix('scylla-bench')
```

Comment:
this change is not related to have sh or not.
it's depend on having an entrypoint set to what.

I rather align this to set explicitly the entry point SCT expect, then modifying the command, based on unrelated logic